int64[] id
int64[] timestep
string[] category
float32[] xpos
float32[] ypos
float32[] velocity
float32[] direction
float32[] omega
float32[] interest